docs/scarpet/api/Auxiliary.md

```diff
@@ -142,6 +142,33 @@ Available shapes:
      * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
      * `level` - level of details, see `'sphere'`.
 
+ * `'poly'`:
+   * Required attributes:
+     * `points` - list of points defining vertices of the polygon
+   * Optional attributes:
+     * `relative` - list of bools. vertices of the polygon that affected by 'follow'. Default means that every point is affacted.
+     * `mode` - how those points are connected. may be "polygon"(default),"strip" or "triangles". "polygon" means that it will be viewed as vertices of a polygon center on the first one. "strip" means that it will be viewed as a triangles strip. "triangles" means that it will be viewed as some triangles that are not related to each other (therefor length of `points` in this mode have to be a multiple of 3).
+     * `inneredges` - if `true` it will make the inner edges be drawn as well. 
```

Comment:
inner? (options should be as short and clear skipping unimportant details, like 'edges')

Reply:
yes it is inner edges indeed.

said if i draw a square with inner edge, it will be like 〼..... the polygon is made by triangles. if inner edge is on, edges of triangles will be drawn as well. that is usefull when those triangles are not in the same plane.

Reply:
and if that is off. only outer edge are drawn. that is usefull when those triangles are in the same plane to creat a polygon.

Reply:
> inner? (options should be as short and clear skipping unimportant details, like 'edges')

i renamed it to inner.